### PR TITLE
🌐 Add Japanese translation for Project Generation

### PR DESCRIPTION
**Pull request by tokusumi**
This PR translates the project-generation.md to Japanese.

@ryuckel @SwftAlpc @komtaki @Attsun1031 and others
Feel free to comment your advise or approve.

### CODE REVIEW

File: docs/ja/docs/project-generation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# プロジェクト生成 - テンプレート
+
+プロジェクトジェネレーターは、初期設定、セキュリティ、データベース、初期APIエンドポイントなどの多くが含まれているため、プロジェクトを開始するために利用できます。
```
Comment (komtaki) [inline docs/ja/docs/project-generation.md:3]:
```suggestion
プロジェクトジェネレーターは、初期設定、セキュリティ、データベース、初期APIエンドポイントなどの多くが含まれているため、プロジェクトの開始に利用できます。
```

「ため」が連続しており、日本語として違和感を感じました。 🤔

File: docs/ja/docs/project-generation.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    * **JWT認証**の処理。
+    * ログインビュー。
+    * ログイン後の、メインダッシュボードビュー。
+    * メインダッシュボードでのユーザー作成と編集。
+    * セルフユーザー版
+    * **Vuex**。
+    * **Vue-router**。
+    * 美しいマテリアルデザインコンポーネントのための**Vuetify**。
+    * **TypeScript**。
+    * **Nginx**ベースのDockerサーバ (Vue-routerとうまく協調するような構成)。
```
Comment (komtaki) [inline docs/ja/docs/project-generation.md:45]:
```suggestion
    * **Nginx**ベースのDockerサーバ (Vue-routerとうまく協調する構成)。
```

「ような」はない方が読みやすいし、なくてもニュアンスは変わらない気がします。 :thinking:

File: docs/ja/docs/project-generation.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## spaCyとFastAPIを使用した機械学習モデル
+
+GitHub: <a href="https://github.com/microsoft/cookiecutter-spacy-fastapi" class="external-link" target="_blank">https://github.com/microsoft/cookiecutter-spacy-fastapi</a>
+
+### spaCyとFastAPIを使用した機械学習モデル - 機能
+
+* **spaCy** のNERモデルの統合。
+* **Azure Cognitive Search** のリクエストフォーマットを搭載。
+* **Production ready** なUvicornとGunicornを使用したPythonウェブサーバ。
```
Comment (komtaki) [inline docs/ja/docs/project-generation.md:81]:
```suggestion
* **リリース可能な** UvicornとGunicornを使用したPythonウェブサーバ。
```

「フルスタック FastAPI PostgreSQL」の「Production Ready」は翻訳されているので、こちらも翻訳して統一した方がいい気がします。

**Review by komtaki — COMMENTED** [review]
Thank you for the translate work. :+1:
I commented to make it easier to read as Japanese.

**Comment by tokusumi** [comment]
@komtaki 
your feedback is helpful. Thank you.

**Comment by komtaki** [comment]
I'm glad to hear that.

LGTM 👌

**Comment by tokusumi** [comment]
please approve this if you ended up reviewing.

**Review by komtaki — APPROVED** [review]

File: docs/ja/docs/project-generation.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+* 完全な**Docker**インテグレーション (Dockerベース)。
+* Docker Swarm モードデプロイ。
+* ローカル開発環境向けの**Docker Compose**インテグレーションと最適化。
+* UvicornとGunicornを使用した**リリース可能な** Python web サーバ。
+* Python <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">**FastAPI**</a> バックエンド:
+    * **高速**: **NodeJS** や **Go** 並みのとても高いパフォーマンス (Starlette と Pydantic のおかげ)。
+    * **直感的**: 素晴らしいエディタのサポートや <abbr title="自動補完、インテリセンスとも呼ばれる">補完。</abbr> デバッグ時間の短縮。
+    * **簡単**: 簡単に利用、習得できるようなデザイン。ドキュメントを読む時間を削減。
+    * **短い**: コードの重複を最小限に。パラメータ宣言による複数の機能。少ないバグ。
```
Comment (Attsun1031) [inline docs/ja/docs/project-generation.md:21]: 原文は `Short: Minimize code duplication. Multiple features from each parameter declaration.` で、「少ないバグ」というのはなくて良いかなと思いました。

**Review by Attsun1031 — COMMENTED** [review]
Thank you for your work 👍 
Please check my review comment.

**Comment by tokusumi** [comment]
@Attsun1031 
thank you for your proposal.

**Review by Attsun1031 — APPROVED** [review]
LGTM 👍

**Comment by tokusumi** [comment]
Hi, @tiangolo
This PR has been approved by two reviewers (github actions to label approved PR didn't work, though).
Would you like to require something more for this to be merged?

**Comment by tiangolo** [comment]
Excellent, thank you @tokusumi ! :clap: :coffee: 

And thanks @Attsun1031 and @komtaki for the reviews! :bow: :cake:

**Pull request merged**